network-api/networkapi/wagtailpages/pagemodels/customblocks/image_feature_block.py

```diff
@@ -0,0 +1,24 @@
+from wagtail.core import blocks
+from wagtail.images.blocks import ImageChooserBlock
+
+
+class ImageFeatureBlock(blocks.StructBlock):
+    image = ImageChooserBlock()
+
+    altText = blocks.CharBlock(
```

Comment:
What do you think about changing this to snake case? 
```suggestion
    alt_text = blocks.CharBlock(
```